Add profile sync prerequisites in the Visual Studio Code guide (#373)
migtools/mta-documentation: docs/topics/vscode/proc_vscode-analyzing-application-using-profiles.adoc

.Prerequisites

* You opened a project written in one of the supported languages in your Visual Studio Code workspace. {ProductShortName} can analyze source code written in `Java`, `C#`, `.NET`, `Go`, and `JavaScript`.
* An architect has configured an analysis profile in the {ProductShortName} web console.

> * The repository URL of the local application must match an existing application in the {ProductShortName} web console.

Reviewer: To preserve the parallel structure required for lists
```suggestion
* The repository URL of the local application matches an existing application in the {ProductShortName} web console.
```

> * The application has a tag that matches with a criteria tag in an archetype.

Reviewer: ```suggestion
* The application has a tag that matches a criteria tag in an archetype.
```

> * The repository URL of the application opened in the Visual Studio Code workspace must match the application repository URL when you configured the application in the {ProductShortName} web console.

Reviewer 2: The repository URL of the local application must match an existing application in the Hub would be more accurate. The current sentence somehow implies that the person working with the application locally in VSCode is the same that configured it in the Hub, which is likely not going to be the case.

> * The application has a tag that matches with an archetype tag in the web console.

Reviewer 2: I would live this like "the application is associated to an Archetype" or "the application matches an Archetype". There is no such thing as an "archetype tag", there are "criteria tags" in Archetypes that drive their association with applications.

* The archetype has at least one target profile.

> * You selected the analysis profile in the target profile. 

Reviewer 2: This is confusing. It is not "you" (with you meaning the person working in VSCode) whom associated the analysis profile in the Hub, it was the Architect. I think the perspective is essential here, as the overarching narrative for this feature is "Developers working with the VSCode extension seamlessly consume the configuration that Architects manage and put together in the Hub".

.. If you want to connect to your Hub without verifying the local SSL certificate in the host, enable *Skip SSL certificate verification*. 
.. Log in to the Hub by enabling the *Enable authentication* switch.
.. Enter the username and password credentials of the {ProductShortName} user interface.

> .. To use the Solution Server to suggest resolutions for the issues in the source code, enable *Solution Server*.

Reviewer: ```suggestion
.. To generate code resolutions for issues in your source code, enable *Solution Server*.
```

.. To sync the profiles with the Hub, enable *Profile Sync*. {ProductShortName} periodically downloads the latest version of the profiles from the Hub. 
+
{ProductShortName} does not update the profiles and rules if you do not sync the profiles with the Hub. 
+

> When you enable the profile sync and the administrator has deployed LLM proxy in the cluster, {ProductShortName} uses the proxy service to connect to the LLM.

Reviewer: Please, just check the tech accuracy for the second paragraph. 
```suggestion
To update your profiles and rules, you must synchronize them with the Hub.
+
When you enable synchronization, {ProductShortName} connects to the large language model (LLM). If an administrator deployed the LLM proxy, the extension routes connections through this service.
```

Author: At the moment, I have not changed the second paragraph much since that would require an SME review to make sure any change is accurate. However, I have moved it as a point under the previous step that speaks about enabling the Solution Server :+1:
